content/docs/about/style-guide.md

```diff
@@ -0,0 +1,323 @@
+the page.
+
+Check the other pages if you're unsure about a particular convention. 
+Examples:
+
+* Most pages in the Kubeflow docs use a period at the end of every list item.
+* There is no period at the end of the page subtitle and the subtitle need not 
+  be a full sentence. (The subtitle comes from the `description` in the front
+  matter of each page.)
+
+## Use active voice rather than passive voice
+
+Passive voice is often confusing, as it's not clear who should perform the
+action.
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Use active voice</th>
+        <th>Instead of passive voice</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>You can configure Kubeflow to</td>
+        <td>Kubeflow can be configured to</td>
+      </tr>
+      <tr>
+        <td>Add the directory to your path</td>
+        <td>The directory should be added to your path</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Use simple present tense
+
+Avoid future tense ("will") and complex syntax such as conjunctive mood 
+("would", "should").  
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Use simple present tense</th>
+        <th>Instead of future tense or complex syntax</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>The following command provisions a virtual machine</td>
+        <td>The following command will provision a virtual machine</td>
+      </tr>
+      <tr>
+        <td>If you add this configuration element, the system is open to
+          the Internet.</td>
+        <td>If you added this configuration element, the system would be open to
+          the Internet.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+**Exception:** Use future tense if it's necessary to convey the correct
+meaning. This requirement is rare.
+
+## Address the audience directly
+
+Using "we" in a sentence can be confusing, because the reader may not know
+whether they're part of the "we" you're describing. For example, compare the
+following two statements:
+
+* "In this release we've added many new features."
+* "In this tutorial we build a flying saucer."
+
+The words "the developer" or "the user" can be ambiguous. For example, if the
+reader is building a product that also has users, then the reader does not
+know whether you're referring to the reader or the users of their product.
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Address the reader directly</th>
+        <th>Instead of "we", "the user", or "the developer"</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Include the directory in your path</td>
+        <td>The user must make sure that the directory is included in their path.
```

Comment:
s/path./path

Please consider removing this full stop, or adding full stops to the other examples in this table.

Reply:
Done.